src/app/pages/HomePage/HomePage.tsx

```diff
@@ -45,7 +45,6 @@ const HomePage = ({ pageData }: HomePageProps) => {
     translations,
     product,
     serviceLocalizedName,
-    homePageTitle,
```

Comment:
Is this used anywhere else? If not, should we consider removing it entirely from the Service Context + service files?